.gitignore

```diff
@@ -148,7 +148,7 @@ celerybeat.pid
 
 # Environments
 .env
-.venv
+.venv*
```

Comment:
My only critique here is that this `.gitignore` is the GitHub-standard, so this deviates from it. Should we add this separately? Or, is it even necessary?

Reply:
github names them templates rather than standards and they do have a lot of content that doesn't apply to us, i normally only added something to gitignore once we were using it